tests/dummy/config/environment.js

```diff
@@ -16,7 +19,9 @@ module.exports = function(environment) {
     APP: {
       // Here you can pass flags/options to your application instance
       // when it is created
-    }
+    },
+
+    ACCEPTANCE_TESTS: fs.readdirSync(path.join(__dirname, '..', 'app', 'acceptance', 'tests'))
```

Comment:
I don't know if this applies to add-ons, but I feel like this should be behind an environment flag.